content/en/docs/getting-started/prerequisites/hardware/snp.md

```diff
@@ -8,5 +8,32 @@ tags:
 - SEV
 ---
 
-TODO
```

Comment:
I think you can drop your first commit in this PR and rebase. The platform setup stuff is already upstream.